nf_core/sync.py

```diff
@@ -64,7 +73,8 @@ def __init__(
     ):
         """Initialise syncing object"""
 
-        self.pipeline_dir = os.path.abspath(pipeline_dir)
+        self.pipeline_dir = Path(pipeline_dir).absolute()
+        self.repo = ensure_git_reop(self.pipeline_dir)
```

Comment:
```suggestion
        self.repo = ensure_git_repo(self.pipeline_dir)
```

Reply:
The suggestions seem not to change anything.

@mashehu Was there a different version here when the changes were proposed?

Reply:
and by version i mean a different function name...